lib/jambots/renderers/cli_renderer.rb

```diff
@@ -3,8 +3,19 @@
 require "tty-spinner"
 require "pastel"
 
-module Jambots
-  class Renderer
+module Jambots::Renderers
+  class CliRenderer
+    def render(conversation, &block)
+      spinner.auto_spin
```

Comment:
I think the spinner should be put in the Thor file, and these renderers are like Rails views, where everything is computed and they just render the final response.

Just an idea

Reply:
I have some doubts about it. Could we have problems if we have a spinner and create a not-pretty output to send it in a pipeline like in the following example?

```
$ jambots chat "hello" --no-pretty  > hello.txt
```

Reply:
What happens right now when you do this? I did that in the Kevin/Bob experiment and I didn't see any spinners. I suppose they buffer play TTY does avoids weird behaviors when redirecting the output.

If this is covered it should be OK

Reply:
Ok, I'll try it